site/en/tutorials/load_data/images.ipynb

```diff
@@ -108,15 +112,12 @@
       "metadata": {
         "colab": {},
         "colab_type": "code",
-        "id": "QGXxBuPyKJw1"
+        "id": "3vhAMaIOBIee"
       },
       "outputs": [],
       "source": [
-        "try:\n",
-        "  # %tensorflow_version only exists in Colab.\n",
-        "  !pip install tf-nightly\n",
-        "except Exception:\n",
-        "  pass\n",
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
```

Comment:
The futures should be above the pip installs. We need this to make it work internally too :)

Reply:
This isn't consistent across the set, for example: https://github.com/tensorflow/docs/blob/master/site/en/tutorials/quickstart/advanced.ipynb

Reply:
Ohh.. well.. LGTM